templates/boot/generic-qemu-boot-template.jinja2

```diff
@@ -19,6 +23,11 @@
 {% endblock %}
 {% block actions %}
 
+{% if qemu_docker %}
+tags:
+    - qemu-docker
+{% endif %}
+
```

Comment:
Is this tag is necessary?
thanks!

Reply:
Without this tag, labs which have workers with qemu AND does not support docker will get baseline-qemu-docker jobs and thoses jobs will fail.

Reply:
I think this should be done via the KernelCI YAML config, to filter jobs that get scheduled in labs, rather than rely on some arbitrary tags to be created in all the LAVA labs.